docs/index.md

```diff
@@ -1121,6 +1121,25 @@ Options:
 
 The SoftLayer driver will use `UBUNTU_LATEST` as the image type by default.
 
+#### Joyent SmartDataCenter
+
+Create machines on [Joyent's SmartDataCenter](http://joyent.com/).
+
+Before using the Docker service on Joyent's SmartDataCenter, you need to have completed the signup process and generated a set of SSH keys (make sure you know the name and location of your SSH key). Please visit [getting started](https://www.joyent.com/developers/getting-started) for more information.
+
+If you want to try to run SDC on your laptop please visit [the getting started with Cloud on a Laptop section](https://github.com/joyent/sdc#getting-started) or if you already are running SmartDataCenter [here](https://github.com/joyent/sdc-docker) is what you need to do in order to enable the Docker service
+
+        $ docker-machine create --driver sdc --sdc-region=$REGION --sdc-account=$ACCOUNT --sdc-key=$PATH_TO_SSH_KEY
```

Comment:
Perhaps this would be preferable?

```bash
docker-machine create --driver triton —triton-dc=$DATA_CENTER —triton-account=$ACCOUNT —triton-key=$PATH_TO_PRIVATE_SSH_KEY